### ISSUE TITLE
'genetic_alg' does not work for multiple layers

### ISSUE THREAD

**Issue by furuolan** [gkhayes/mlrose#23]
**From the tutorial (revised) hidden_nodes = [5,5,5,5,5]**

np.random.seed(3)

nn_model1 = mlrose.NeuralNetwork(hidden_nodes = [5,5,5,5,5], activation ='relu', 
                                 algorithm ='genetic_alg', 
                                 max_iters = 1000, bias = True, is_classifier = True, 
                                 learning_rate = 0.0001, early_stopping = True, 
                                 clip_max = 5, max_attempts = 100)

nn_model1.fit(X_train_scaled, y_train_hot)

Error message:
ValueError: Input contains NaN, infinity or a value too large for dtype('float64').

**Comment by geoff-va** [comment]
You can try reducing the `max_clip` value to something much smaller. Start at 1 and work your way up.

There is a softmax in the GA calculation and the weights do seem to blow up quickly in GA. I don't know if this is actually a 'bug' in any way, or just a consequence of the algorithm.

**Comment by furuolan** [comment]
Thanks so much!

**Comment by geoff-va** [comment]
@gkhayes Hi Genevieve - first, great job on this package! It's really nice and intuitive, and we all appreciate all the work you put into it! 

Might you have any insight into what's going on here? I'm using a [24, 24] network with RELU and with GD i've got to stick to a learning rate of ~0.001 or smaller and clip of 1.5 to get it to run at all. Otherwise the `softmax` call overflows and by the time we get to `log_loss` call it flags it for containing `NaN`'s. I have normalized the inputs (though I didn't use the `MinMaxScaler`,  but the `StandardScaler` instead.

The min/max values I get for `x` coming into the `softmax` are:
```
Min: -44.48151729485336
Max: 68.1884513917445

1st iteration
Min: -7280.53099075823
Max: 7280.53099075823
```

I have similar problems using GA. Hill climbing and annealing work fine.

Thanks!

**Comment by zemerica** [comment]
I'm having the same issue as well - I wasn't sure if it had something to do with one-hot-encoded multiclassification problems?

**Comment by B-Kopp** [comment]
I am having the same issue. I tried normalizing the softmax function to dampen it's output but that did not fix the problem.

**Comment by mjschock** [comment]
this should be resolved in #25.  lemme know if that fix is not working for you
